src/screens/LocationPage/LocationPage.tsx

```diff
@@ -17,6 +17,8 @@ function WithSearchInNav({ region }: LocationPageProps) {
   let { chartId } = useParams<{ chartId: string }>();
   const [menuOpen, setMenuOpen] = useState(false);
   const hasScrolled = useShowPastPosition(850);
+  const [yPosition, setYposition] = useState(0);
+  window.addEventListener('scroll', () => setYposition(window.scrollY));
```

Comment:
Is there a reason we're putting this way up on the `LocationPage` component and then passing the `yPosition` prop down through all the components?  I think that means every time you scroll, React is going to re-render a bunch of components since the props will have changed.  I think putting the scroll listener as close to where we need (CountyMap?) might make more sense.

Also, I think this should be inside a hook that removes the listener when the hook is removed.  As-is, we're going to end up registering more and more of these listeners and they'll never get removed.

See the `useShowPastPosition()` hook for something similar. https://github.com/covid-projections/covid-projections/blob/develop/src/common/hooks/useShowPastPosition.ts#L5